fsharp-backend/src/BackendOnlyStdLib/LibHttpBytesClient.fs

```diff
@@ -0,0 +1,84 @@
+/// v5 StdLib functions in the HttpClient module
+module BackendOnlyStdLib.LibHttpBytesClient
+
+open System.Net.Http
+
+open Prelude
+open LibExecution
+open LibExecution.RuntimeTypes
+
+module Errors = LibExecution.Errors
+
+let incorrectArgs = Errors.incorrectArgs
+
+let fn = FQFnName.stdlibFnName
+
+// Converts an object to (string,string) pairs. Raises an exception if not an object
+// TODO: consider if a _Tuple_ should be expected for request headers
+let toStringPairs (dv : Dval) : Result<List<string * string>, string> =
+  match dv with
+  | DList tpls ->
+    tpls
+    |> List.map (fun pair ->
+      match pair with
+      | DTuple (DStr k, DStr v, []) -> Ok(k, v)
+      | wrong ->
+        Error
+          $"Expected a (string * string), but got: {DvalReprDeveloper.toRepr wrong}")
+    |> Tablecloth.Result.values
+  | _ -> Error $"Expected a tuple, but got: {DvalReprDeveloper.toRepr dv}"
+
+// todo: move this to the consuming file probably
+let call (method : HttpMethod) =
+  (function
+  | _, [ DStr uri; DBytes body; query; headers ] ->
+    let headers = toStringPairs headers
+    let query = HttpQueryEncoding.toQuery query
+    match headers, query with
+    | Ok headers, Ok query ->
+      HttpBytesClient.sendRequest uri method body query headers
+    | _ -> incorrectArgs ()
+  | _ -> incorrectArgs ())
+
+/// Used in both Ok and Error cases
+let headersType = TList(TTuple(TStr, TStr, []))
+let queryStryingType = TList(TTuple(TStr, TStr, []))
+
+let responseType =
+  TRecord [ "body", TBytes; "headers", headersType; "code", TInt; "error", TStr ]
+
+let returnTypeOk = responseType
+let returnTypeErr = responseType
+let returnType = TResult(returnTypeOk, returnTypeErr)
```

Comment:
"error" probably doesn't fit in `returnTypeOk`, right? I forget why it's included for existing httpclient OK responses.